community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPageCache.java

```diff
@@ -573,30 +573,30 @@ public void flushAndForce() throws IOException
     }
 
     @Override
-    public synchronized void flushAndForce( IOLimiter limiter ) throws IOException
+    public void flushAndForce( IOLimiter limiter ) throws IOException
     {
         if ( limiter == null )
         {
             throw new IllegalArgumentException( "IOLimiter cannot be null" );
         }
         assertNotClosed();
-        flushAllPages( limiter );
+        List<PagedFile> files = listExistingMappings();
```

Comment:
Yes, see the commit I just pushed. It's technically _"fine"_ because the check pointer will retry, but it is obviously inconveniencing the tests. It also pointed to some optimisation opportunities that @burqen will look at in due time.